todoApp/samples/dotnet/Program.cs

```diff
@@ -56,6 +61,34 @@
     Console.WriteLine($"Item title: {i.Title}, status: {i.Status}");
 }
 
+// create item via multipart/form-data
+using var todoItemsContent = new MultiPartFormDataBinaryContent();
+var newItem = new TodoItem("Read code", TodoItemStatus.NotStarted);
+todoItemsContent.Add(ModelReaderWriter.Write(newItem), "items", contentType: "application/json");
+const string codeFilename = "note1.txt";
+await using var codeFileStream = File.OpenRead(codeFilename);
```

Comment:
Does file open/read depends on where the code is run?

Reply:
oh so you remind me that I did not push csproj change.
Just did it: https://github.com/allenjzhang/typespec-e2e-demo/pull/79/commits
thank you!